integrationtests/tests/fixtures/fake.rs

```diff
@@ -46,8 +46,9 @@ impl FakeLightningTest {
     }
 }
 
+#[async_trait]
 impl LightningTest for FakeLightningTest {
```

Comment:
Looking at this made me wonder why do we even have `LightningTest` trait ? If its only too add functions that are not part of a rpc interface (but an addition) we might want to switch that to the `LnRpcAdapter` and add the functions to it @jkitman ?

Reply:
The reason was I didn't want to add functions to a trait that are only used for testing.

Reply:
oh that makes sense